Library/Formula/jenkins.rb

```diff
@@ -58,12 +64,12 @@ def caveats; <<-EOS.undent
   test do
     ENV["JENKINS_HOME"] = testpath
     pid = fork do
-      exec "#{bin}/jenkins"
+      exec "java -Djava.io.tmpdir=#{testpath} -jar #{opt_libexec}/jenkins.war"
```

Comment:
Since normal usage doesn't require staying inside the sandbox, I think we should leave the value alone except during the test.